### DISCUSSION TITLE
I can not do it. pls help

### DISCUSSION THREAD

**Discussion by madboy31** [Discord-OTP-Forcer/Discord-OTP-Forcer#141] [Community Support]
I dont know about Python things but i tried to understand a bit from youtube and ur instruction/tutorial but it doesnt work. I did click setup.bat then i did fill up infos :

Basic login info
email: "myemail@hotmail.com"  ✔
password: ""   (here's empty as u see because i dont remember my password but i tried pu somethings too but still doesnt work)  ✔

 Program mode configuration
Lets you skip the initial mode prompt stage by filling in your desired
programMode and codeMode beforehand.
programMode: "reset"  ✔
codeMode: "normal"  ✔

Password Reset Info

To get your token it will be in the url of the password reset page.
Follow the instructions in the README.md on how to obtain this token
newPassword: "somerandompassord"  ✔
resetToken: "myresettoken"  ✔

Disable 2FA info
Fill in your discord account token here.
authToken: ""  (idk here, i think must be empty idk actually so i didnt touch)

Hide SENSITIVE_DEBUG log info
Default = True (didnt touch)
sensitiveDebug: "True" (didnt touch)

Log file creation
Default = False  (didnt touch)
logCreation: "False"  (didnt touch)

in cfg.yml  then i save then program says click any key for continue then i do thennnnn i click start.bat then program again says click any key  then i do that too but no reaction. program closing itself and nothing happen. please help me.

**Comment by Inbydev** [comment]
@madboy31 could you try with the latest commit?

https://github.com/Derpitron/Discord-OTP-Forcer/archive/refs/heads/main.zip

**Comment by Inbydev** [comment]
This probably is an Issue of the past, the program got a lot of updates since then, so re open this discussion if you encounter the same issue with the latest release!

https://codeberg.org/Discord-OTP-Forcer/Discord-OTP-Forcer/releases/tag/v0.5.0-beta

And follow the tutorial:

https://discord-otp-forcer.codeberg.page/en/
